### PR TITLE
Split server url definition into separate file.

### PR DESCRIPTION
**Pull request by eyurko**

### CODE REVIEW

File: client/app.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <script src="../third_party/codemirror-5.19.0/addon/edit/matchbrackets.js"></script>
     <script src="../third_party/ui-codemirror-0.3.0/ui-codemirror.min.js"></script>
     <script src="app.js"></script>
+    <script src="server_url.js"></script>
```
Comment (seanlip) [inline client/app.html]: Perhaps the new file should be named serverUrl.js to conform with all the others (like checkBalancedParentheses.js)?

File: client/server_url.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/**
```
Comment (seanlip) [inline client/server_url.js]: Copyright notice?

File: client/server_url.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+ * @fileoverview Adds a SERVER_URL constant to point to TIE's current server.
+ */
+
+tie.constant('SERVER_URL', null);
```
Comment (seanlip) [inline client/server_url.js]: Should we remove the corresponding constant from app.js?

File: client/app.html (hunk 1 above)
Comment (eyurko) [inline client/app.html]: aw dangit, you right.

I've been hanging out in python for too long.

Fixing now.

File: client/server_url.js (hunk 3 above)
Comment (eyurko) [inline client/server_url.js]: Oh, I thought I did that. Weird.

File: client/server_url.js (hunk 2 above)
Comment (eyurko) [inline client/server_url.js]: Oh, yes, I'll add that. Sorry.

**Pull request merged**